kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/common/create-domain-job.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright 2018, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+
+# Include common utility functions
+source ${CREATE_DOMAIN_SCRIPT_DIR}/utility.sh
+
+# Verify the script to create the domain exists
+script=${CREATE_DOMAIN_SCRIPT_DIR}/create-domain-script.sh
+
+checkCreateDomainScript $script
+checkDomainSecret
+prepareDomainHomeDir
+
+# Execute the script to create the domain
+source $script
+
+# Create the node manager
```

Comment:
Maybe be a 'TBD' note here that this step can be removed when the NM bootstrapping rework is merged

Reply:
Similarly, the same note can go in setup-nodemanager.py

Reply:
The potential cleanup work is tracked by OWLS-69133.